### ISSUE TITLE
iPad Pro BrowserView and MobileView not working properly

### ISSUE THREAD

**Issue by upalatucci** [duskload/react-device-detect#146]
On iPad Pro iOS 14 - Safari BrowserView and MobileView render their children at the same time.
It seems not to work properly.
Is there anybody working on this? Can I take it?

**Comment by skoal2007** [comment]
Yes, I have same issue on my ipad air 4th, chrome works fine, but Safari render both  BrowserView and MobileView. Any solution?

**Comment by skoal2007** [comment]
Hi Upalatucci, 

As the people mentions on other thread, you can solve this issue with the work around solution 
```
{!isTablet && <BrowserView/> }
```

**Comment by upalatucci** [comment]
Thanks man!
